src/control/directive.spec.ts

```diff
@@ -560,6 +560,113 @@ describe(NgrxFormControlDirective.name, () => {
     });
   });
 
+  describe('suppression of global action emit', () => {
+    it('be disabled by default', () => {
+      expect(directive.ngrxSuppressGlobalActionEmit).toBe(false);
+      directive.ngOnInit();
+      expect(directive.ngrxSuppressGlobalActionEmit).toBe(false);
+    });
+
+    describe('is disabled', () => {
```

Comment:
aren't these tests redundant since they represent the existing behavior that is already tested?